modules/osu_library_hero/osu_library_hero.module

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * Implements hook_modules_installed().
+ *
+ * Once the Default content is imported we need to update the media_id for the
+ * background image. This hook is called after every module is installed.
+ * By checking for the module type and to ensure we are not in configuration
+ * import step:
+ *   - Load the media entity by UUID and get its ID.
+ *   - Load the section library by UUID and update its settings by setting
+ *     the media_id and save it back to the database.
+ */
+function osu_library_hero_modules_installed($modules, $is_syncing) {
+  if (in_array('osu_library_hero', $modules) && !$is_syncing) {
+    /** @var \Drupal\media\MediaStorage $media_storage */
+    $media_storage = \Drupal::entityTypeManager()->getStorage('media');
+    /** @var array $media_item_arr */
+    $media_item_arr = $media_storage->loadByProperties(['uuid' => 'c28c4077-c0ba-4f08-8ab3-a2397ff108c6']);
+    /** @var \Drupal\media\Entity\Media $media_item */
+    $media_item = reset($media_item_arr);
+    $media_id = $media_item->get('mid')->first()->getValue()['value'];
+    /** @var \Drupal\Core\Entity\Sql\SqlContentEntityStorage $section_storage */
+    $section_storage = \Drupal::entityTypeManager()
+      ->getStorage('section_library_template');
+    /** @var array $hero_section_arr */
+    $hero_section_arr = $section_storage->loadByProperties(['uuid' => '93f31c1f-c230-4449-a8cd-f5918aeb2853']);
+    /** @var \Drupal\section_library\Entity\SectionLibraryTemplate $hero_section */
+    $hero_section = reset($hero_section_arr);
+    /** @var Drupal\layout_builder\Plugin\Field\FieldType\LayoutSectionItem $layouts */
+    $layouts = $hero_section->get('layout_section')->first();
+    /** @var \Drupal\layout_builder\Section $layout_section */
+    $layout_section = $layouts->getValue()['section'];
+    $layout_section_settings = $layout_section->getLayoutSettings();
```

Comment:
new line here as well

Reply:
I branched off from the hero for the two column so these have been fixed in the hero branch.